stable_baselines3/her/her_replay_buffer.py

```diff
@@ -78,16 +77,15 @@ def _sample_transitions(self, batch_size: int) -> ReplayBufferSamples:
         t_samples = np.array([np.random.choice(np.arange(ep_len)) for ep_len in episode_lengths])
         # get selected timesteps
         transitions = np.array([buffer[ep][trans] for ep, trans in zip(episode_idxs, t_samples)], dtype=object)
-        # get her samples indices with her_ratio
-        her_idxs = np.where(np.random.uniform(size=batch_size) < self.her_ratio)[0]
+        # get her samples indices with her_prob
+        her_idxs = np.where(np.random.uniform(size=batch_size) < self.her_prob)[0]
```

Comment:
maybe take a look at `np.random.choice()` because with `her_ratio` and the batch size, you can know how many HER samples you will need, no?

something like (please double check):

```python
her_indices = np.random.choice(np.arange(batch_size), int(self.her_ratio * batch_size), replace=False)
# equivalent to:
her_indices = np.random.permutation(np.arange(batch_size))[:int(self.her_ratio * batch_size)]

```

Reply:
or do you need `her_prob` somewhere else?